mw-webapp/src/hooks/useLoad.ts

```diff
@@ -0,0 +1,75 @@
+import {useEffect, useState} from "react";
+
+/**
+ * Use Load hook Props
+ */
+interface useLoadProps<T, D> {
```

Comment:
we can use real words in generics, T and D mean nothing in the current context. try to rename them, I hope it could increase readability a lot)

Reply:
Renamed